tests/functional/retry/test_retry.py

```diff
@@ -134,7 +134,7 @@ def test_run_operation(self, project):
         )
 
         expected_statuses = {
-            "operation.test.alter_timezone": RunStatus.Error,
+            "macro.test.alter_timezone": RunStatus.Error,
```

Comment:
this is a breaking change from the user perspective - unless it's an intentional change we want to make or its strictly necessary for implementation, i think we should avoid making it.

Reply:
I don't have a strong opinion on whether the unique_ids should look like `operation.*` or `macro.*`. I'm not sure what the context is but if there was some rationale for opting for `operation.*` when we introduced this initially, that probably still holds.